functions.tex

```diff
@@ -101,7 +103,8 @@ \chapter{Functions and Passing types}
 int* i, j; // Warning: this does not mean what you think
 int *k; // the * can also go against the variable name
 int*p; // also legal. C doesn't care about spaces here
-int * q; // Now you're just being silly. (Still legal though).
+int * q; // Now you're just being silly. (Still legal though)
+int** m; // Pointers to pointers are okay too
 \end{codeblock}
 The last two lines are examples of legal code which is unnecessarily hard to read.
```

Comment:
No longer last 2 since you added the ** example, probably just move m above the badly spaced ones